client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -73,377 +73,466 @@ module.exports = protectForm( React.createClass( {
 	},
 
 	resetState: function() {
-		var clearSettings = { fetchingSettings: true };
+		const clearSettings = { fetchingSettings: true };
 		this.discussionAttributes.forEach( function( attribute ) {
 			clearSettings[ attribute ] = '';
 		} );
 		this.replaceState( clearSettings );
 		debug( 'resetting state' );
 	},
 
-	defaultArticleSettings: function() {
+	setDirtyField( key ) {
```

Comment:
`state.dirtyFields` is still used by [form-base](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/form-base.js#L52) to check whether the input value should be updated (a "dirty" field has been touched by the user, and should not update even if the server value changed). What we removed was the linked value, not the concept of dirty fields.